plugins/globalias/globalias.plugin.zsh

```diff
@@ -0,0 +1,11 @@
+globalias() {
+   zle _expand_alias
+   zle expand-word
+   zle self-insert
+}
+zle -N globalias
+bindkey -e " " globalias
+bindkey -v " " globalias
+bindkey -e "^ " magic-space           # control-space to bypass completion
```

Comment:
The fact that you do the `bindkey -v` makes it use the vicmd keymap. Use `bindkey -M` instead:
- `bindkey -M vicmd` instead of `bindkey -v`
- `bindkey -M emacs` instead of `bindkey -e`